### DISCUSSION TITLE
Xarray User Forum at the 2021 Dask Distributed Summit

### DISCUSSION THREAD

**Discussion by andersy005** [pydata/xarray#5294] [Announcements]
Please join us for the 2021 Xarray User Forum at the Dask Distributed Summit on May 21, 4pm - 6pm UTC / 12pm - 2pm US/Eastern. This interdisciplinary event will bring together Xarray users from different scientific domains and Xarray developers.

The highlight of this forum will be presentations and discussions on (1) Xarray+Dask user stories from non-geoscience domains (2) updates on new and upcoming features of Xarray.

For the full program, and the list of speakers, please see
https://summit.dask.org/schedule/presentation/51/xarray-user-forum/

You can register for the Dask Distributed Summit at: https://summit.dask.org/tickets/

**Comment by abel-bzz** [comment]
Just putting the link to the youtube video of the talk for people coming here: https://www.youtube.com/watch?v=BXSjGgHtAX4
